### PR TITLE
added test.js

### PR DESCRIPTION
**Pull request by after12am**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+a
```
Comment (after12am) [inline test/test.js]: what is this?

Comment (okami-satoshi) [inline test/test.js]: I don't know.

File: test/test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+b
```
Comment (okami-satoshi) [inline test/test.js]: wow!

File: test/test.js (hunk 1 above)
Comment (okami-satoshi) [inline test/test.js]: second time

File: test/test.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+c
+
+d
+
+e
+
+f
+
+g
```
Comment (okami-satoshi) [inline test/test.js]: message after tweak

File: test/test.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+a
+
+b
+
+c
+
+d
+
+e
+
```
Comment (after12am) [inline test/test.js]: oh no!

File: test/test.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+f
```
Comment (after12am) [inline test/test.js]: come on!

File: test/test.js (hunk 1 above)
Comment (after12am) [inline test/test.js]: ok

Comment (after12am) [inline test/test.js]: ok

Comment (after12am) [inline test/test.js]: ok

**Comment by after12am** [comment]
hey Yo!

File: test/test.js (hunk 1 above)
Comment (after12am) [inline test/test.js]: ok

**Pull request closed without merging**